openpype/hosts/blender/utility_scripts/make_paths_absolute.py

```diff
@@ -25,11 +25,11 @@
         nargs="?",
         help="source filepath",
     )
-    args = parser.parse_args(sys.argv[sys.argv.index("--") + 1 :])
+    args = parser.parse_args(sys.argv[sys.argv.index("--") + 1:])
```

Comment:
I think we better do:
```suggestion
    args = parser.parse_known_args(sys.argv[sys.argv.index("--") + 1:])
```